pkgs/build-support/fetchurl/mirrors.nix

```diff
@@ -305,13 +279,10 @@
   postgresql = [
     "http://ftp.postgresql.org/pub/"
     "ftp://ftp.postgresql.org/pub/"
-    "ftp://ftp-archives.postgresql.org/pub/"
   ];
 
   metalab = [
-    "ftp://mirrors.kernel.org/metalab/"
     "ftp://ftp.gwdg.de/pub/linux/metalab/"
```

Comment:
I think we want another mirror here.

Reply:
finding one was not easy. I still don't know if there is an official mirror list anywhere.